six/topshot/six-topshot-transfer-moment/src/six-topshot-transfer-moment.js

```diff
@@ -0,0 +1,81 @@
+import * as fcl from "@onflow/fcl"
+import * as t from "@onflow/types"
+import {config} from "@onflow/config"
+
+const DEPS = new Set([
+    "0xTOPSHOT",
+    "0xNONFUNGIBLETOKEN"
+])
+
+export const TITLE = "NBA Top Shot Transfer Moment"
+export const DESCRIPTION = "Transfers a moment from an authorizer's NBA Top Shot collection to another account's."
+export const VERSION = "0.0.1"
+export const HASH = "3bb66424f129bee4605ef2f932ce8c133385beea019518f316bae7a5c34aa7bd"
+export const CODE = 
+`import NonFungibleToken from 0xNONFUNGIBLETOKEN
+import TopShot from 0xTOPSHOT
+
+// This transaction transfers a moment to a recipient
+
+// This transaction is how a topshot user would transfer a moment
+// from their account to another account
+// The recipient must have a TopShot Collection object stored
+// and a public MomentCollectionPublic capability stored at
+// \`/public/MomentCollection\`
+
+// Parameters:
+//
+// recipient: The Flow address of the account to receive the moment.
+// withdrawID: The id of the moment to be transferred
+
+transaction(recipient: Address, withdrawID: UInt64) {
+
+    // local variable for storing the transferred token
+    let transferToken: @NonFungibleToken.NFT
+    
+    prepare(acct: AuthAccount) {
+
+        // borrow a reference to the owner's collection
+        let collectionRef = acct.borrow<&TopShot.Collection>(from: /storage/MomentCollection)
+            ?? panic("Could not borrow a reference to the stored Moment collection")
+        
+        // withdraw the NFT
+        self.transferToken <- collectionRef.withdraw(withdrawID: withdrawID)
+    }
+
+    execute {
+        
+        // get the recipient's public account object
+        let recipient = getAccount(recipient)
+
+        // get the Collection reference for the receiver
+        let receiverRef = recipient.getCapability(/public/MomentCollection).borrow<&{TopShot.MomentCollectionPublic}>()!
```

Comment:
No need to check or `panic` here if recipient is missing Collection or Capability?

Reply:
The force unwrap `!` at the end of the line will cause the script to abort if the borrow'd reference is nil https://docs.onflow.org/cadence/language/values-and-types/#force-unwrap-